app/components/new-project-modal/component.js

```diff
@@ -0,0 +1,73 @@
+import { computed } from '@ember/object';
+import { A } from '@ember/array';
+import { inject as service } from '@ember/service';
+import Component from '@ember/component';
+import { task, timeout } from 'ember-concurrency';
+
+export default Component.extend({
+    store: service(),
+    newNode: null,
+    more: false,
+    institutionsSelected: computed.oneWay('user.institutions'),
```

Comment:
You are right about the oneWay.  A solution is to reset institutionsSelect when selecting one. We should optimize for one or two institutions. I would be surprised if more than a handful of users have more than one institution, and if any have more than 2, since to make be a member one needs to have an email address for that institutions and/or login through their portal, then merge those accounts.